internal/datanode/importv2/task_l0_import_test.go

```diff
@@ -191,7 +191,7 @@ func (s *L0ImportSuite) TestL0Import() {
 
 	deltaLog := actual.GetBinlogs()[0]
 	s.Equal(int64(s.delCnt), deltaLog.GetEntriesNum())
-	s.Equal(s.deleteData.Size(), deltaLog.GetMemorySize())
```

Comment:
why is the assert removed

Reply:
The serialized data sizes are not the same due to differing implementation. It is no longer a requirement that the two sizes here are equal.